src/node.cc

```diff
@@ -2566,6 +2566,7 @@ static void PrintHelp() {
          "  --max-stack-size=val set max v8 stack size (bytes)\n"
          "  --enable-ssl2        enable ssl2\n"
          "  --enable-ssl3        enable ssl3\n"
+         "  --enable-small-dh-groups  enable small dh groups (not recommended)\n"
```

Comment:
Not recommend options are not only `--enable-small-dh-groups` but also `-enable-ssl2` and `-enable-ssl3`. I wonder if we need to show explicitly its recommendation in the help message.